src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Let's make sure we don't use disable linter rules. These are here to help us write clean code.

Reply:
I disable because the error the linter rule was telling me to create a purê funcional component, and the requirement asks for another thing.